lib/matplotlib/quiver.py

```diff
@@ -392,6 +392,12 @@ def _parse_args(*args):
     return X, Y, U, V, C
 
 
+def check_array_shapes(*arrays):
```

Comment:
Can you make this function private? I don't think it should be part of our public API.

I also think the name of the function is not very explicit on what it does. How about `check_consistent_shape`?